### ISSUE TITLE
If you want to clip grad, insert this...

### ISSUE THREAD

**Issue by zeng-hello-world** [suvojit-0x55aa/mixed-precision-pytorch#1]
```python

torch.nn.utils.clip_grad_norm_(self.master_params, max_norm=5*self._LOSS_SCALE)

```

**Comment by lijie2160** [comment]
> ```python
> torch.nn.utils.clip_grad_norm_(self.master_params, max_norm=5*self._LOSS_SCALE)
> ```

how to add this operation, please.

**Comment by suvojit-0x55aa** [comment]
This can be done right after `loss.backward()` and must be done before an `optimizer.step()`
